### PR TITLE
Update footer label in reader for "Current Chapter"

### LINKED ISSUE #357: Change default "chapter" string for unknown chapters
At the very least, we should titlecase this and come up with something less ambiguous.

### PR DESCRIPTION
**Pull request by gioneill**
Updating default in spineItemDict to title-cased localized string "Current Chapter".

fixes #357

### CODE REVIEW

**Comment by gioneill** [comment]
Changed base branch to 'develop'

File: Simplified/NYPLReaderReadiumView.m [hunk 1]
```diff
@@ -811,7 +811,7 @@ - (void)calculateBookLength
         [spineItemDict setObject:title forKey:@"tocElementTitle"];
       }
       else {
-        [spineItemDict setObject:NSLocalizedString(@"chapter", nil) forKey:@"tocElementTitle"];
+        [spineItemDict setObject:NSLocalizedString(@"ReaderViewControllerCurrentChapter", nil) forKey:@"tocElementTitle"];
```
Comment (aferditamuriqi) [inline Simplified/NYPLReaderReadiumView.m]: this key exists, right? i haven't checked , just making sure we have the actual localized string for this.

Comment (gioneill) [inline Simplified/NYPLReaderReadiumView.m]: Yes it's already been translated too.

**Comment by aferditamuriqi** [comment]
👍🏻

File: Simplified/NYPLReaderReadiumView.m (hunk 1 above)
Comment (aferditamuriqi) [inline Simplified/NYPLReaderReadiumView.m]: 👍🏻

**Pull request merged**